api/handlers.py

```diff
@@ -0,0 +1,71 @@
+import webapp2
+from google.appengine.ext.webapp import template
+from loveutils import messages
+from loveutils import config
+
+
+MESSAGE_TEMPLATE_PATH = 'templates/message.html'
+HOME_TEMPLATE_PATH = 'templates/home.html'
+MESSAGES = messages.MESSAGES
+
+
+def handle_404():
```

Comment:
Instead of duplicating that function, could you import the one you defined in `main.py`?